src/components/migrationToV3.stories.mdx

```diff
@@ -62,18 +66,21 @@ const MyApp = () => {
     const response = await fetch('/my-meetings-endpoint');
     const data = await response.json();
     const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
+    const logger = new ConsoleLogger('SDK', LogLevel.WARN);
     const eventController = new DefaultEventController(
-        meetingSessionConfiguration, 
-        meetingManager.createLogger(meetingSessionConfiguration), 
-        new NoOpEventReporter()
-      );
-    const joinData = {
-      meetingSessionConfiguration: meetingSessionConfiguration,
-      eventController: eventController
+      meetingSessionConfiguration, 
+      logger, 
+      new NoOpEventReporter()
+    );
+    const options = {
+      eventController
```

Comment:
I see in the quickStarts, and most places we end an object property with a `,`. Could you please follow the same in this migration guide as well?

Reply:
This was missed in my review from Levi's PR. Will check and fix later.